common/plugins/eu.esdihumboldt.hale.common.instance.orient/src/eu/esdihumboldt/hale/common/instance/orient/storage/StoreInstancesJob.java

```diff
@@ -120,6 +128,10 @@ public IStatus run(IProgressMonitor monitor) {
 			// use intent
 			db.declareIntent(new OIntentMassiveInsert());
 
+			final InstanceProcessingExtension ext = new InstanceProcessingExtension(
+					serviceProvider);
+			final List<InstanceProcessor> processors = ext.getInstanceProcessors();
```

Comment:
An improvement for later could be that we are more flexible regarding which processors are actually run (e.g. here only those that are required for the transformation). For now, and as this is also running in the UI, I think it is best to run all as you did here.